examples/ngrok-http-full.js

```diff
@@ -35,17 +35,14 @@ builder
   })
   .handleUpdateCommand((update) => {
     console.log(
-      "update command, version: " +
-        update.version +
-        " permitMajorVersion: " +
-        update.permitMajorVersion
+      `update command, version: ${update.version} permitMajorVersion: ${update.permitMajorVersion}`
     );
   })
   .handleHeartbeat((latency) => {
-    console.log("heartbeat, latency: " + latency + " milliseconds");
+    console.log(`heartbeat, latency: ${latency} milliseconds`);
   })
   .handleDisconnection((addr, error) => {
-    console.log("disconnected, addr: " + addr + " error: " + error);
+    console.log(`disconnected, addr: ${addr} error: ${error}`);
```

Comment:
`console.log` also takes multiple args that it prints as a single space separated string to the console if that is appealing.

```js
console.log("disconnected, addr:", addr, "error:",  error);
```

Reply:
heh, always so many ways to do the same thing. Comma does seem to win in length and clarity when the message ends in a variable, which is the vast majority of cases here, so switched to commas.